### ISSUE TITLE
Creating a One-Page for Community-Building Resources

### ISSUE THREAD

**Issue by bnb** [nodejs/community-committee#42]
Someone just asked in the Education repo (https://github.com/nodejs/education/issues/31) if they could start a meetup at their school. I absolutely _love_ the initiative and thought here, and would like to see if there's some kind of document we could pull together with resources that are ready-to-use like NodeSchool.

I've had a hard time with NodeSchool myself, but that's because I've always gone solo - would be great to pull similar resources, groups, and efforts into a One Page that can be easily consumed and shared.

**Comment by fforres** [comment]
It's a nice idea 😸 
There is an organizers wiki for Nodeschool here. https://github.com/nodeschool/organizers/wiki

You wan't something different? Less `nodeschool-workshop` and more `meetup-y`?

**Comment by mikeal** [comment]
This is a great idea, we should pull together resources like like the NodeSchool guide into one big resource page :)

**Comment by gautamarora** [comment]
@bnb i can offer to help with this. From what I understand the goal here is to create a one-pager that lists out ideas & resources for someone interested in running a node-focussed event in their community.

Rough outline could be something like....

```
Interested in starting a Node event for your community, here are some ideas:

  1. [NodeSchool - series of workshops on javascript, node and more](https://github.com/nodeschool/organizers/wiki)
  2. [NodeTodo - self-serve contributing to node](http://nodetodo.org/)
  3. [Code & Learn - contributing to node with attendance of ](https://nodejs.org/en/get-involved/code-and-learn/)
  4. Meetups - talks about the node ecosystem
  ...
```

**Comment by bnb** [comment]
@gautamarora Yes! That sounds awesome. Would absolutely LOVE to see a doc based on that outline.

**Comment by bnb** [comment]
@gautamarora Are you still interested in working on this? If so, we'd love to have your help! If not, absolutely zero worries - what you've shared already is a super helpful place for us to get started. 🙏

**Comment by pup** [comment]
i'd be very interested in helping out for this too! let me know if you need any help or resources from the community committee @gautamarora

**Comment by gautamarora** [comment]
@bnb @pup - thanks for checking in on this. Apologies for the delay, I wont be able to get to this in jan so please go ahead in creating this doc.

**Comment by bnb** [comment]
Thanks for initially offering to help on this @gautamarora - much appreciated, and no worries about not having time. ❤️

@pup do you want to take this on, or maybe do a call-out in the next CommComm meeting to request participation?

**Comment by flickz** [comment]
Being the individual who asked for this on [Nodejs/Education](https://github.com/nodejs/education/issues/31) I will like to help with the doc. @gautamarora already provided 3 links to resources. Any suggestion on other links?

**Comment by pup** [comment]
@bnb i'd love to get some more hands to work on this, so i think i'll bring it up in the next meeting

**Comment by bnb** [comment]
@flickz I'd suggest checking out a prior attempt at this in the [Evangelism WG](https://github.com/nodejs/evangelism/blob/master/documents/meetup-starter-kit.md) and the [Local Initiatives Directory](https://github.com/nodejs/evangelism/tree/master/documents/local_initiatives) in the same repo 😄

**Comment by flickz** [comment]
@bnb Able to combine some ideas from [Local Initiatives Directory](https://github.com/nodejs/evangelism/edit/master/documents/local_initiatives/README.md). Here is a rough doc [Community Building Resources](https://github.com/flickz/community-committee/blob/master/documents/Community-Building-Resources.md). Feedback?

**Comment by chowdhurian** [comment]
Is there any activity currently happening around this issue? 🙂

**Comment by Tiriel** [comment]
I'm going to second @chowdhurian 's call on activity here!
